packages/standards/biome-esm.jsonc

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "linter": {
+    "domains": {
+      "react": "all",
+    },
+    "rules": {
+      "nursery": {
+        "noGlobalDirnameFilename": "error",
+      },
+      "performance": {
+        // Namespace imports (`import * as foo from 'foo'`) can severely impact performance and
+        // tree-shaking capabilities in various bundlers. We had a prior rule within our
+        // `@readme/eslint-plugin` package, `readme/no-wildcard-imports`, that enforced this. This
+        // is the Biome equivalent.
+        //
+        // https://biomejs.dev/linter/rules/no-namespace-import/
+        "noNamespaceImport": "error",
+      },
+      "style": {
+        "noCommonJs": "error",
```

Comment:
Previously enabled in `@readme/eslint-config/esm` via [import/no-commonjs](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-commonjs.md).